src/Agent.Sdk/ContainerInfo.cs

```diff
@@ -243,11 +258,15 @@ public void ExpandProperties(Variables variables)
             // Expand image and options strings
             ContainerImage = variables.ExpandValue(nameof(ContainerImage), ContainerImage);
             ContainerCreateOptions = variables.ExpandValue(nameof(ContainerCreateOptions), ContainerCreateOptions);
-        }
+        }*/
     }
 
     public class MountVolume
     {
+        public MountVolume()
+        {
+
+        }
```

Comment:
it concerns me that this was needed

Reply:
nm, I now understand why this is needed. It is needed because an instance of ContainerInfo which contains instances of MountVolume was added to an object that is serialized to another process.